utils/stacktrace/should-ignore.js

```diff
@@ -22,14 +22,15 @@ const errorsToIgnore = [
   'stop_youtube',
   'null%20is%20not%20an%20object%20(evaluating%20%27elt.parentNode%27)',
 ];
+const JS_REGEX = /\.(mjs|js(\.br)?)$/;
```

Comment:
We can remove the `.br` stuff, brotli URLs are no longer used by the Cache (it was a temporary hack).

The duplicate `endsWith` checks might be marginally faster, but a simple `\.m?js$` would be fine.

Reply:
Done.